lib/malan_web/views/session_view.ex

```diff
@@ -25,7 +25,10 @@ defmodule MalanWeb.SessionView do
       valid_only_for_ip: session.valid_only_for_ip,
       valid_only_for_approved_ips: session.valid_only_for_approved_ips,
       location: session.location,
-      is_valid: Accounts.session_valid_bool?(session.expires_at, session.revoked_at)
+      is_valid: Accounts.session_valid_bool?(session.expires_at, session.revoked_at),
+      extendable_until: session.extendable_until,
+      max_extension_secs: session.max_extension_secs,
+      extensions: session.extensions
```

Comment:
actually yeah I thought about that too and I think it could become an issue if the session has a long history of extensions.  The data is all embedded as JSON inside the session object, so there aren't other queries, so the main consideration is network transfer.  In an ideal I would create an endpoint for querying these and make it paginated, but the scope increase made me weary.  I also considered adding a query param that will include them so that way if the caller doesn't need the extension history, it isn't fetched every time.  In the end I decided it would be easy enough to do either of those implementations as a second pass, but actually that would break the contract, so maybe that's not a good idea...

I'll add a query param switch so at least we have non-breaking options in the future.  Does that sound ok?